packages/components/src/ListingSummary/NewsroomInfo.tsx

```diff
@@ -1,13 +1,18 @@
 import * as React from "react";
 import { ListingSummaryComponentProps } from "./types";
-import { NewsroomIcon, StyledListingSummaryTop, StyledListingSummaryNewsroomName } from "./styledComponents";
+import {
+  NewsroomIcon,
+  NewsroomLogo,
+  StyledListingSummaryTop,
+  StyledListingSummaryNewsroomName,
+} from "./styledComponents";
 import DepositOrStakeAmount from "./DepositOrStakeAmount";
 import NewsroomTagline from "./NewsroomTagline";
 
 const NewsroomInfo: React.SFC<ListingSummaryComponentProps> = props => {
   return (
     <StyledListingSummaryTop>
-      <NewsroomIcon />
+      <NewsroomIcon>{props.logoURL && <NewsroomLogo src={props.logoURL} />}</NewsroomIcon>
```

Comment:
Should we have a placeholder logo?

Reply:
Good call I'll ask Nguyet - as it is currently, `<NewsroomIcon>` draws a blank grey box, so spacing is the same and it'll look okayish next to other logos, but yeah a default would be good.